src/hooks/useActiveNetwork.ts

```diff
@@ -245,8 +245,10 @@ export function useActiveNetwork() {
         })
         history.push(target)
       } catch (switchError) {
+        // This is a workaround solution for Coin98
+        const isSwitcherror = typeof switchError === 'object'
```

Comment:
should check empty here
`typeof switchError === 'object' && Object.keys(switchError).length === 0`